input/pagecontent/changes.md

```diff
@@ -23,7 +23,7 @@ To help implementers, only the more significant changes are listed here.
   </li>
   <li>New value sets:
   <ul>
-  <li><a href="ValueSet-rsg-source-document-jurisdiction.html">AU Recorded Sex or Gender (RSG) Source Document Jurisdiction</a></li>
+  <li><a href="valueset-au-jurisdiction-extended.html">Jurisdiction - AU Extended</a></li>
```

Comment:
IG publisher isn't recognising the url with that case. Replace with "ValueSet-au-jurisdiction-extended.html"

Reply:
Updated valueset page name to Title case